helper_test.go

```diff
@@ -174,26 +175,29 @@ func createUploadedConfigurationVersion(t *testing.T, client *Client, w *Workspa
 		t.Fatal(err)
 	}
 
+	WaitUntilStatus(t, client, cv, ConfigurationUploaded, 15)
+
+	return cv, cvCleanup
+}
+
+// helper to wait until a configuration version has reached a certain status
+func WaitUntilStatus(t *testing.T, client *Client, cv *ConfigurationVersion, desiredStatus ConfigurationStatus, timeoutSeconds int) {
```

Comment:
One minor suggestion - I don't think we need to export the function to other packages, so it could instead be `waitUntilStatus`